migrations/migrate.cfg

```diff
@@ -0,0 +1,25 @@
+[db_settings]
+# Used to identify which repository this database is versioned under.
+# You can use the name of your project.
+repository_id=ea restaurant migrations
```

Comment:
Could you avoid using white spaces? Use underscore instead

Reply:
done